app/views/devise/registrations/new.html.erb

```diff
@@ -1,28 +1,32 @@
-
 <div class="container">
-  <h2>Sign up</h2>
+  <div class="row justify-content-center">
+    <div class="col-lg-6">
```

Comment:
What about md? I think we could use 8 there.

Reply:
yes